### PR TITLE
Verify that all tests files are imported.

### PR DESCRIPTION
**Pull request by Raynos**
This PR adds a script that verifies that all tests are
imported. It makes a few assumptions

 - all files in test/**/*.js are test files that contain
    `tape` blocks.
 - any files in lib/*.js are non-test files that contain
    NO `tape` blocks.
 - All tests are imported from `test/index.js`.
 - All tests are imported as `require(./PATH.js)`

I've had to inline a couple of folder imports into the
test/index.js for this to work. I've also had to move
a few helper files into lib folder.

I've found two test files that were never imported and
made them pass.

r: @kriskowal @shannili @rf

### CODE REVIEW

**Comment by kriskowal** [comment]
:ok:

**Comment by ShanniLi** [comment]
lgtm

File: node/scripts/verify_included.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+# set -x
+
+FILES=$(
+    ls --color=never test/**/*.js test/*.js | \
```
Comment (kriskowal) [inline node/scripts/verify_included.sh]: This line is an error on “BSD”. illegal option -- -.

**Comment by kriskowal** [comment]
I tacked another commit on to make this work on “BSD”. Rebase if you wish.

**Pull request merged**